test_runner/src/main/kotlin/ftl/run/platform/android/CreateAndroidTestContext.kt

```diff
@@ -39,13 +39,14 @@ private suspend fun List<AndroidTestContext>.setupShards(
 ): List<AndroidTestContext> = coroutineScope {
     map { testContext ->
         async {
+            val newArgs = if (testContext is InstrumentationTestContext && testContext.maxTestShards != null) {
+                args.copy(commonArgs = args.commonArgs.copy(maxTestShards = testContext.maxTestShards))
+            } else args
```

Comment:
could we pack it in some method 
A. `fun AndroidArgs.prepareArgsForSharding(): AndroidArgs`
B. `fun getInstrumentationArgs(args: AndroidArgs): AndroidArgs`
C: some other naming